js/src/admin/components/MailPage.js

```diff
@@ -108,6 +105,28 @@ export default class MailPage extends Page {
     );
   }
 
+    getMailSettingsElementForm(field) {
+
+    if (field === 'mail_mailgun_endpoint') {
+
+      let endpoints = {
+        'api.eu.mailgun.net': 'EU Endpoint',
+        'api.mailgun.net': 'US Endpoint',
```

Comment:
These values should be translatable.

Reply:
Please check the current implementation.